src/textual/widgets/_select.py

```diff
@@ -19,6 +20,11 @@
 
     from ..app import ComposeResult
 
+if sys.version_info.minor >= 11:
+    from typing import Self
+else:
+    from typing_extensions import Self
+
 
```

Comment:
```suggestion

```

Reply:
Thanks.  I've resolve this in the latest commit